pkg/template/master_template.go

```diff
@@ -111,6 +111,26 @@ systemd:
         ash -c "cat /etc/kubernetes/config/kubelet.yaml.tmpl |envsubst >/etc/kubernetes/config/kubelet.yaml"
       [Install]
       WantedBy=multi-user.target
+  - name: k8s-setup-apiserver-manifest.service
+    enabled: true
+    contents: |
+      [Unit]
+      Description=k8s-setup-apiserver-manifest Service
+      After=docker.service
+      Requires=docker.service
+      [Service]
+      Type=oneshot
+      RemainAfterExit=yes
+      TimeoutStartSec=0
+      Environment=IMAGE={{ .Images.Envsubst }}
+      ExecStartPre=/opt/bin/setup-apiserver-environment
+      ExecStart=docker run --rm \
+        --env-file /etc/apiserver-environment \
+        -v /etc/kubernetes/:/etc/kubernetes/ \
+        $IMAGE \
+        ash -c "cat /etc/kubernetes/manifest-templates/k8s-api-server.yaml.tmpl |envsubst >/etc/kubernetes/manifests/k8s-api-server.yaml"
```

Comment:
Typo?

```suggestion
        sh -c "cat /etc/kubernetes/manifest-templates/k8s-api-server.yaml.tmpl |envsubst >/etc/kubernetes/manifests/k8s-api-server.yaml"
```

Reply:
Also `/etc/kubernetes/manifest-templates/k8s-api-server.yaml.tmpl` is `/etc/kubernetes/manifest/k8s-api-server.yaml.tmpl`

Reply:
nope, `ash` is `alpine shell` and it has a bit more features than plain `sh`

Reply:
Ah right 👍🏻